code/render/batching.cpp

```diff
@@ -884,6 +894,16 @@ void batching_add_tri(int texture, vertex *verts)
 	batching_add_tri_internal(batch, texture, verts);
 }
 
+void batching_add_tri(int texture, vertex *verts, primitive_batch *batch)
+{
+	Assertion((texture >= 0), "batching_add_...() attempted for invalid texture");
+	if ( texture < 0 ) {
+		return;
+	}
+
+	batching_add_tri_internal(batch, texture, verts);
+}
```

Comment:
So ... what does this function do? Extra checking?

Reply:
Both of these were inherited from #5154's change sets, I didn't realize they'd still show up here. I've rebased on the merge commit on there and these have dropped out of the github change view, since they aren't actually changes.